File: src/StealME.Client/src/stealme/client/communication/PC.java
package stealme.client.communication;

public class PC {

	//public final static char LOGICAL_MESSAGE_SEPARATOR = '|';
	// message prefixes
	public final static String COMMAND_PREFIX = "CMD";
	public final static String MESSAGE_PREFIX = "MSG";
	public final static String GET_PREFIX = "GET";
	public final static String SET_PREFIX = "SET";

	// command types
	public final static String COMMAND_ACTIVATE = "ACTIVATE";
	public final static String COMMAND_DEACTIVATE = "DEACTIVATE";
	public final static String COMMAND_SIGNAL = "SIGNAL";

	// set types
	public final static String SET_MODE = "MODE";
	public final static String SET_INTERVAL = "INTERVAL";

	// get types
	public final static String GET_STATUS = "STATUS";
	public final static String GET_IMEI = "IMEI";

	// message types
	public final static String MESSAGE_IMEI = "IMEI";
	public final static String MESSAGE_TCP_LOCATION = "TCP";
	public final static String MESSAGE_LOCATION = "LOC";
	public final static String MESSAGE_STATUS = "STAT";
	public final static String MESSAGE_STATE = "STATE";

	// mode types
	public final static String MODE_ACTIVATED = "ACTIVATED";
	public final static String MODE_DEACTIVATED = "DEACTIVATED";
	public final static String MODE_ALARM = "ALARM";
}

File: src/StealME.Server/StealME.Server.Messaging/Interfaces/PC.cs
namespace StealME.Server.Messaging.Protocol
{
    using System;

    public class PC
    {
        public const char LOGICAL_MESSAGE_SEPARATOR = '|';

        //message prefixes
        public const String COMMAND_PREFIX = "CMD";
        public const String MESSAGE_PREFIX = "MSG";
        public const String GET_PREFIX = "GET";
        public const String SET_PREFIX = "SET";

        //command types
        public const String COMMAND_ACTIVATE = "ACTIVATE";
        public const String COMMAND_DEACTIVATE = "DEACTIVATE";
        public const String COMMAND_SIGNAL = "SIGNAL";

        //set types
        public const String SET_MODE = "MODE";
        public const String SET_INTERVAL = "INTERVAL";

        //get types
        public const String GET_STATUS = "STATUS";
        public const String GET_IMEI = "IMEI";

        //message types
        public const String MESSAGE_IMEI = "IMEI";
        public const String MESSAGE_TCP_LOCATION = "TCP";
        public const String MESSAGE_LOCATION = "LOC";
        public const String MESSAGE_STATUS = "STAT";
        public const String MESSAGE_STATE = "STATE";

        //mode types
        public const String MODE_ACTIVATED = "ACTIVATED";
        public const String MODE_DEACTIVATED = "DEACTIVATED";
        public const String MODE_ALARM = "ALARM";
    }
}
